### PR TITLE
Content head tweaks on Amp

### PR DESCRIPTION
**Pull request by zeftilldeath**
Before
screen shot 2015-09-23 at 17 40 55

After
screen shot 2015-09-23 at 17 40 36

### CODE REVIEW

**Comment by zeftilldeath** [comment]
There is a (.u-h) star rating (1 / 5 stars) alongside the stars. Not sure what it's for. I've temporarily hidden it with display: none

**Comment by gidsg** [comment]
@zeftilldeath I think the stuff with class `.u-h` is supposed to be off screen but visible to screenreaders.

**Comment by NataliaLKB** [comment]
@zeftilldeath Looks good, but you are going to have to merge with master.

File: common/app/views/fragments/amp/stylesheets/main.scala.html [hunk 1]
```diff
@@ -54,7 +54,6 @@
         border-bottom: 0.0625rem dotted #dfdfdf;
```
Comment (johnduffell) [inline common/app/views/fragments/amp/stylesheets/main.scala.html]: why have all the file modes changed from 644 to 755?  I think this is wrong, they should be executable

Comment (desbo) [inline common/app/views/fragments/amp/stylesheets/main.scala.html]: yep, you can tell git to ignore permissions changes with `git config core.fileMode false`

**Comment by NataliaLKB** [comment]
Once the issue that @johnduffell raised is dealt with: :+1: . Everything else looks great.

**Pull request closed without merging**